sql/sql_load.cc

```diff
@@ -1450,7 +1465,9 @@ read_xml_field(THD *thd, COPY_INFO &info, TABLE_LIST *table_list,
     }
     
     WSREP_LOAD_DATA_SPLIT(thd, table, info);
-    if (write_record(thd, table, &info))
+    err= write_record(thd, table, &info);
```

Comment:
`write_record()` returns `int`, not `bool`. Change the type of `err`.
I’d even narrow the scope of `err` and declare it here:
```c++
int err= write_record(thd, table, &info);
table->auto_increment_field_not_null= FALSE;
if (err)
```
On the other hand, if you follow my RAII suggestion, this change should not be needed at all.